tests/integration/test_services_integration.py

```
@@ -498,6 +505,41 @@ async def test_create_and_cancel_reminder(self, db_schema, tenant_id, async_sess
+        svc = NotificationService(async_session)
+
+        # Create user and notification under tenant 1
+        uid1 = await self._seed_user(tenant_id, async_session)
+        await svc.send_notification(
+            user_id=uid1, notification_type="info", title="T1", content="m", tenant_id=tenant_id
+        )
+
+        # Create user under tenant 2
+        suffix = uuid.uuid4().hex[:8]
+        user_svc = UserService(async_session)
+        uid2 = uid2.id
```

Pylint (4.0.5)

[error] Using variable 'uid2' before assignment

(E0601)